lexicons/tools/ozone/set/upsertSet.json

```diff
@@ -0,0 +1,24 @@
+{
+  "lexicon": 1,
+  "id": "tools.ozone.sets.upsertSet",
```

Comment:
We totally don't have to use it, but figure it's worth mentioning that we use the name `repo.putRecord` for upserting records into a repo.